t/docs/02-tutorial-suites.t

```diff
@@ -25,14 +25,18 @@
 # `etc/tutorial/NAME` directory.
 #-------------------------------------------------------------------------------
 . $(dirname $0)/test_header
-skip_all "skipped: @TODO: Awaiting App upgrade to Python3"
 #-------------------------------------------------------------------------------
 # Generate list of tests.
 TEST_KEYS=('')
 TUT_DIRS=('')
 TESTS=('')
 TUTORIALS_PATH="${ROSE_HOME}/etc/tutorial"
+SKIPS=('')  # TODO - remove me once Python3 conversion complete
```

Comment:
It'd make me happy if you'd record on this or #2317 whether you want to 
* Wait on rose suite-run update on PR #2317 to be merged, rebase and remove this
* Create an issue so we know that it's a TODO, or add  a note to my PR?

Reply:
Can do either way, I don't mind.

Added skips for now just for the niceness of having the tests go green.

Reply:
I'd favour option 1. One less PR over-all.